src/ex2/app.js

```diff
@@ -0,0 +1,124 @@
+import { ItemManager } from "./itemManager.js"
+import { PokemonClient } from "./pokemonClient.js"
+
+const addButton = document.getElementById("list-item-submit")
+const textInput = document.getElementById("list-item-input")
+const allLists = document.getElementById("list")
+const counterLists = document.getElementById("counter")
+const deleteAllLists = document.getElementById("deleteAllTasks")
+const errorMessage = document.getElementById("error")
+const regex = /^[0-9]*$/
+
+const itemManager = new ItemManager()
+const pokemonClient = new PokemonClient()
+class Main {
+  constructor() {
+    this.name = "Main"
+  }
+  init() {
+    addButton.addEventListener("click", function () {
+      validation()
+    })
+  }
+}
+
+function addNewList(name) {
```

Comment:
you should created a single render method that will get the list from the item manger and re-render the whole list, that would get make your code clearer, easier to maintain, and you'll have only one function that can "draw" the whole list no matter if you delete an item, cleared them all or added 4 new items
<img width="425" alt="image" src="https://user-images.githubusercontent.com/65164924/170869392-c3d48b84-5472-4ad9-8be1-b4bc7723d219.png">